src/components/numbers/Numbers.js

```diff
@@ -1,19 +1,23 @@
 import './numbers.scss';
 
 const Numbers = (props) => {
-  const { from } = props;
-  const { to } = props;
-  const { even } = props;
+  const { from, to, even } = props;
   const arr = [];
 
-  for (let num = from; num <= to; num++) {
+  for (let num = +from; num <= +to; num++) {
```

Comment:
`from` and `to` should be as numbers, and if it is a number you don't need to use `+`